### PR TITLE
Add Team JaM's 2nd blog post

### PR DESCRIPTION
**Pull request by malisas**
@jeenalee and I wrote the second blog post for our team. We are scheduled for the 2016-09-28 publish date.

### CODE REVIEW

File: blog/_posts/2016-09-28-almost-done-jammin.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Almost done jammin'
+layout: post
+created_at: Mon Sep 25 2016
+permalink: blog/2016-09-28-goodbye-jam
```
Comment (alicetragedy) [inline blog/_posts/2016-09-28-almost-done-jammin.md]: could you make the permalink consistent with the file name? (meaning: using the same name for both)

File: blog/_posts/2016-09-28-almost-done-jammin.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Almost done jammin'
+layout: post
+created_at: Mon Sep 25 2016
```
Comment (alicetragedy) [inline blog/_posts/2016-09-28-almost-done-jammin.md]: can you make the created_at date the publishing date (same date as in the filename and permalink).

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @malisas great post 👍 there are just tiny changes to be made but after that's done it's ready to be merged. thanks for submitting on time!

**Comment by malisas** [comment]
Ok, I made the changes and squashed! Hope it's good to go :)

**Review by alicetragedy — APPROVED** [review]
thanks for the changes — it's all good to go!

**Pull request merged**